.github/workflows/integration.yml

```diff
@@ -68,14 +68,46 @@ jobs:
           SNOWFLAKE_ROLE: ${{ secrets.SNOWFLAKE_ROLE }}
         run: make test-acceptance
 
-      - name: comment on pull request
-        if: github.event_name == 'repository_dispatch'
+      - name: find comment
+        if: ${{ always() }}
+        uses: peter-evans/find-comment@v1
+        id: fc
+        with:
+          issue-number: ${{ github.event.pull_request.number }}
+          comment-author: 'github-actions[bot]'
+          body-includes: Integration tests ran for
+
+      - name: create comment
+        if: (${{ failure() }} || ${{ cancelled() }} || ${{ success() }}) && ${{ steps.fc.outputs.comment-id == 0 }} && (github.event_name == 'repository_dispatch')
```

Comment:
huh, surprised all of these failure, cancelled, scucess work but... always doesnt?

Reply:
i'm not sure if it works... but it might? I feel like it's more specific than always so it doesn't just ignore everything else?